<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportFromHEDialogBase</class>
 <widget class="QDialog" name="ImportFromHEDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>472</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QKan Erzeugen von Haltungsflächen</string>
  </property>
  <widget class="QDialogButtonBox" name="button_box">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>400</y>
     <width>281</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Bearbeitung starten ...&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>420</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Erzeugen von Haltungsflächen (Voronoi-Flächen)</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lf_anzahl_haltungen">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>300</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string extracomment="Anzahl der abhängig von den Filteroptionen zu berücksichtigenden Haltungen"/>
   </property>
   <property name="statusTip">
    <string extracomment="Anzahl der abhängig von den Filteroptionen zu berücksichtigenden Haltungen"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QListWidget" name="lw_hal_entw">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>331</width>
     <height>201</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="toolTip">
    <string extracomment="Auswahl der zu bearbeitenden Haltungen nach Entwässerungssystem. Alle auswählen mit [Strg]+A">&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Liste der in den Haltungen verwendeten Entwässerungssysteme./nStrg + Klick ändert Auswahlstatus, Shift + Klick erweitert Auswahlbereich&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="statusTip">
    <string extracomment="Auswahl der zu bearbeitenden Haltungen nach Entwässerungssystem. Alle auswählen mit [Strg]+A">Liste der in den Haltungen verwendeten Entwässerungssysteme. Strg + Klick ändert Auswahlstatus, Shift + Klick erweitert Auswahlbereich</string>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::ExtendedSelection</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>300</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="toolTip">
    <string extracomment="Anzahl der abhängig von den Filteroptionen zu berücksichtigenden ganzen Flächen"/>
   </property>
   <property name="statusTip">
    <string extracomment="Anzahl der abhängig von den Filteroptionen zu berücksichtigenden ganzen Flächen"/>
   </property>
   <property name="text">
    <string>Haltungen</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="cb_selHalActive">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>370</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Aktiviert Auswahl. Wenn deaktiviert, werden alle Entwässerungssysteme berücksichtigt&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="statusTip">
    <string>Aktiviert Auswahl. Wenn deaktiviert, werden alle Entwässerungssysteme berücksichtigt</string>
   </property>
   <property name="text">
    <string>Nur ausgewählte Entwässerungsarten berücksichtigen</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>300</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string extracomment="Anzahl der abhängig von den Filteroptionen zu berücksichtigenden Flächen (mit und ohne Verschneidung)"/>
   </property>
   <property name="statusTip">
    <string extracomment="Anzahl der abhängig von den Filteroptionen zu berücksichtigenden Flächen (mit und ohne Verschneidung)"/>
   </property>
   <property name="text">
    <string>Aktuell betroffen:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lf_anzahl_flaechen">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>330</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string extracomment="Anzahl der abhängig von den Filteroptionen zu berücksichtigenden Haltungen"/>
   </property>
   <property name="statusTip">
    <string extracomment="Anzahl der abhängig von den Filteroptionen zu berücksichtigenden Haltungen"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>330</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="toolTip">
    <string extracomment="Anzahl der abhängig von den Filteroptionen zu berücksichtigenden ganzen Flächen"/>
   </property>
   <property name="statusTip">
    <string extracomment="Anzahl der abhängig von den Filteroptionen zu berücksichtigenden ganzen Flächen"/>
   </property>
   <property name="text">
    <string>Flächen</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lf_warning">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>360</y>
     <width>371</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string extracomment="Anzahl der abhängig von den Filteroptionen zu berücksichtigenden ganzen Flächen">&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Die Voronoiflächen werden nur für Haltungsflächen erzeugt, die aufzuteilende Flächen enthalten. &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="statusTip">
    <string extracomment="Anzahl der abhängig von den Filteroptionen zu berücksichtigenden ganzen Flächen"/>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>accepted()</signal>
   <receiver>ImportFromHEDialogBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>265</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>275</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>ImportFromHEDialogBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>265</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>275</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
